### ISSUE TITLE
Sync fail to Dropbox in Mac 

### ISSUE THREAD

**Issue by astrod** [laurent22/joplin#2131]
## Environment

Joplin version:  1.0.174
Platform: Mac
OS specifcs: 10.14.6(Mojave)

## Steps To Reproduce

1. Click Joplin
2. Select Synchronise
3. Go to Dropbox and get a code
4. Enter the code to Joplin
5. After 2-3 min, error message appear

> Could not authorise application:
>request to https://api.dropboxapi.com/oauth2/token failed, reason: read ECONNRESET
>Please try again.

Describe what you expected to happen:

Sync to Dropbox

**Comment by astrod** [comment]
I builded Joplin to solve this problem. And I find that build fail when node version is 13.2.0. I guess it's because of sqllite. I changed version from 13.2.0 to 10.17.0 and success to build.

Then I guess that your clientId or clientSecret is not vaild. I call this api
~~~
api.dropboxapi.com/2/check/app 
~~~

for test and it looks like failed. It might be helpful to check it.
I hope this is helpful for you.

Thank you.
